### ISSUE TITLE
Local SDK installation broken

### ISSUE THREAD

**Issue by graemerocher** [sdkman/sdkman-cli#664]
We require a conversation to take place in the appropriate Gitter Room before raising a new Github Issue. Please note that **issues will be closed immediately if prior discussion did not take place**. We need to do this to help manage the quality and validity of Issues raised on this project.

Please tick one:
- [ ] this is a new Feature Request, a conversation was started in the [Gitter CLI Development Room](https://gitter.im/sdkman/cli-dev).
- [x ] this is a new Bug Report or Issue, a conversation was started in [Gitter User Issues Room](https://gitter.im/sdkman/user-issues).

Please explain the Issue / Feature Request here:

Local SDK installation is broken:

```bash
sdk install java graal ~/oss/graalvm-ee-1.0.0-rc4/Contents/Home/
Invalid version! graal with length 5 exceeds max of 15!
```

**Comment by marc0der** [comment]
Thanks for letting us know @graemerocher. I'll make work of it asap. Someone recently introduced this validation on the version length, but the [testing](https://github.com/sdkman/sdkman-cli/blob/master/src/test/cucumber/local_developement_versions.feature#L69) in the PR seems a bit skimpy.

**Comment by graemerocher** [comment]
standard bash shell in default Terminal app on OS X

**Comment by marc0der** [comment]
The last fix seems to have done the trick.

I've released it to the beta channel, could you confirm that it's working for you now?

**Comment by graemerocher** [comment]
Seems to work. thx

**Comment by marc0der** [comment]
Released that as version 5.7.2.
